.drone/drone.jsonnet

```diff
@@ -446,7 +446,7 @@ local manifest_ecr(apps, archs) = pipeline('manifest-ecr') {
 
 [
   pipeline('loki-build-image') {
-    local build_image_tag = '0.26.0',
+    local build_image_tag = '0.27.0',
```

Comment:
Was this updated everywhere? https://github.com/grafana/loki/blob/main/docs/sources/maintaining/release-loki-build-image.md has a script to help with this.

Reply:
That needs to be done after this PR lands :slightly_smiling_face: